settings.gradle.kts

```diff
@@ -138,6 +138,8 @@ include(":subprojects:delivery:upload-cd-build-result")
 include(":subprojects:delivery:upload-to-googleplay")
 
 include(":subprojects:emcee:gradle-plugin")
-include(":subprojects:emcee:queue-backend-api")
+include(":subprojects:emcee:queue-client-api")
```

Comment:
The module resctructure.
Modules with `-models` contain only models
Modules with `-api` contain models and retrofit service
`client` related for queue client - gradle-plugin